src/components/Quote.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+import Ninja from './Ninja';
+
+const Quote = () => {
+  const [quote, setQuote] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    fetch('https://api.api-ninjas.com/v1/quotes?category=education', {
+      method: 'GET',
+      headers: { 'X-Api-key': 'CxQhbgnKF2X1iPThUiuulA==2qSxWFNNTyvLkSvt' },
+    })
+      .then((resp) => resp.json())
+      .then((data) => setQuote(data[0].quote))
+      .catch(() => {
+        setHasError(true);
+      });
+    setIsLoading(false);
+  }, [setQuote, setIsLoading]);
+
+  if (hasError) return <div className="displayed-quote">Cant execute!!!There is an error</div>;
+  if (isLoading) return <div className="displayed-quote">Loading...</div>;
+
+  return (
+    <div>
+      <Ninja RandomNinja={`${quote}`} />
+    </div>
+  );
+};
```

Comment:
- [ ] Kindly make sure the `loading` state can be visible on the page when it is `true`.  For now, there is no `Loading...` message appearing when the status is `true`. Kindly fix that to meet the project requirement.

    - ### YOUR APP BEHAVIOUR
    ![chrome-capture-2023-4-7](https://user-images.githubusercontent.com/87197412/236648769-35695568-9025-4866-959e-23f9ad4af5bc.gif)

Reply:
please approve my project I will fix it later I the time is less for this block

Reply:
Hello @SulimanJoya, Kindly check me on Slack instead I can offer guidance on how the error can be solved, please.